third_party/move/move-compiler/tests/move_check/parser/attribute_placement.exp

```diff
@@ -0,0 +1,84 @@
+error[E02015]: invalid attribute
```

Comment:
These tests are expected to pass.

Reply:
I copied the same check to the /skip_attribute_checks/ directory where you can see the old result.